templates/taxi/index.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% block content %}
+    <h1>Taxi Service Home</h1>
+    <p>Welcome!</p>
+    <h3>What we have:</h3>
+    <ul>
+        <li>Amount drivers: {{ num_drivers }}</li>
+        <li>Amount manufacturers: {{ num_manufacturers }}</li>
+        <li>Amount cars: {{ num_cars }}</li>

```

Comment:
Please review the task checklist again: 

Reply:
Please review the task checklist again:

Override block content and place (as a list) information about:
Number of cars
Number of drivers
Number of manufacturers